[DCFS-155] volume has an extra param 'isReady'

Brotholomew opened the pull request:
This additional parameter indicates whether a volume is ready for file operations.

Signed-off-by: Brotholomew <bartoszek.blach@gmail.com>

Brotholomew committed:
[DCFS-155] volume has an extra param 'isReady'

This additional parameter indicates whether a volume is ready for file
operations.

Signed-off-by: Brotholomew <bartoszek.blach@gmail.com>

lavar3l reviewed (changes requested):
Brakuje nam jeszcze sprawdzenia czy volume is ready jak ktoś próbuje wykonać operację na systemie plików (w szczególności upload, download pliku).

Brotholomew committed:
[DCFS-155] adhere to review

1. volume.IsReady() check is performed on file upload request
2. volume is not ready when it does not have any disks

Signed-off-by: Brotholomew <bartoszek.blach@gmail.com>
